wuzz.go

```diff
@@ -166,6 +207,7 @@ func (a *App) Layout(g *gocui.Gui) error {
 		setViewDefaults(v)
 		v.Editable = true
 		v.Title = "URL params (F3)"
+		v.Editor = &singlelineEditor{defaultEditor}
```

Comment:
Yeah but we can't use multilines requests now because <kbd>Enter</kbd> will launch request and will not break lines.